site/_docs/history.md

```diff
@@ -56,20 +132,221 @@ other software versions as specified in gradle.properties.
+  Add ARG_MIN, ARG_MAX (aka MIN_BY, MAX_BY) aggregate functions
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5362">CALCITE-5362</a>]
+  Implement geometry measurement functions
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5393">CALCITE-5393</a>]
+  VALUE as an synonym for VALUES keyword (enabled in MySQL conformance)
+
 #### Dependency version upgrade
 {: #dependency-1-33-0}
 
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5341">CALCITE-5341</a>]
+  Upgrade Calcite to Avatica 1.23.0
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5351">CALCITE-5351</a>]
+  Bump jackson to 2.13.4 and jackson databind to 2.13.4.2 to avoid CVEs
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5356">CALCITE-5356</a>]
+  Update junit4 to 4.13.2 and junit5 to 5.9.1
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5374">CALCITE-5374</a>]
+  Upgrade jackson version to 2.14.0
+
 #### Bug-fixes, API changes and minor enhancements
 {: #fixes-1-33-0}
 
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-4186">CALCITE-4186</a>]
+  Add ST_CoveredBy spatial function
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-4351">CALCITE-4351</a>]
+  RelMdUtil#numDistinctVals always returns 0 for large inputs
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-4632">CALCITE-4632</a>]
+  Exception in RelToSqlConverter: "Cannot convert x to DECIMAL(n, m) due to overflow"
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-4804">CALCITE-4804</a>]
+  Support Snapshot operator serialization and deserialization
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-4972">CALCITE-4972</a>]
+  Subfields of array columns containing structs are not qualified in getFieldOrigins
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-4982">CALCITE-4982</a>]
+  Do not push 'cast to not null' through Join in ProjectJoinTransposeRule
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5127">CALCITE-5127</a>]
+  Support correlation variables in Project
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5141">CALCITE-5141</a>]
+  Incomplete implicit type conversion for insert values
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5159">CALCITE-5159</a>]
+  ARRAY string constructor, and implicit cast from string literal (enabled in PostgreSQL conformance)
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5209">CALCITE-5209</a>]
+  Proper sub-query handling if it is used inside select list and group by
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5217">CALCITE-5217</a>]
+  Implement unparseSqlIntervalLiteral in the Firebolt dialect
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5230">CALCITE-5230</a>]
+  Return type of PERCENTILE_DISC should be the same as sort expression
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5252">CALCITE-5252</a>]
+  JDBC adapter sometimes miss parentheses around SELECT in WITH_ITEM body
```

Comment:
SQL Keywords and Java classes/methods should be enclosed in grave accent chars. Here for instance it should be `SELECT` and `WITH_ITEM`. There are many places where this change should be applied.

Reply:
done

Reply:
How about we add this rule to the [commit message section of our doc](https://github.com/apache/calcite/blob/ae228f64347cc7620d28eff04f5869583c6bf9d5/site/develop/index.md?plain=1#L204-L222)? If the commit messages all obey this rule, this will save the RM's effort, and there is no harm to have the escape char in the commit message IMO.